doc/faq.rst

```diff
@@ -186,26 +187,53 @@ No, you can use symbolic names for messages::
 
 
 4.5 I have a callback function where I have no control over received arguments. How do I avoid getting unused argument warnings?
-----------------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------------------------------
 
 Prefix (ui) the callback's name by `cb_`, as in cb_onclick(...). By
 doing so arguments usage won't be checked. Another solution is to
 use one of the names defined in the "dummy-variables" configuration
 variable for unused argument ("_" and "dummy" by default).
 
 4.6 What is the format of the configuration file?
----------------------------------------------------
+-------------------------------------------------
 
 Pylint uses ConfigParser from the standard library to parse the configuration
 file.  It means that if you need to disable a lot of messages, you can use
-tricks like: ::
+any formatting accepted by ConfigParser, e.g.
+
+.. code-block:: ini
+
+    [MASTER]
+    output-format = colorized
+
+    [Messages Control]
+    disable=method-hidden,too-many-lines,wildcard-import
+
+.. code-block:: ini
 
-    # disable wildcard-import, method-hidden and too-many-lines
-    disable=
-        wildcard-import,
-        method-hidden,
+    [Messages Control]
+    disable =
+        method-hidden
         too-many-lines
+        wildcard-import
+
+Alternatively, if you use ``pyproject.toml``, e.g.
+
+.. code-block:: toml
```

Comment:
Looks like "toml" highlighting is not active for Sphinx, which results in a warning that [makes the build job fail](https://github.com/PyCQA/pylint/runs/6099963981?check_suite_focus=true). Pygments [does support TOML](https://pygments.org/languages/) in general, though, [since version 2.10.0](https://github.com/pygments/pygments/blob/master/CHANGES#L160-L189) (we have [2.11.2 installed](https://github.com/PyCQA/pylint/runs/6099963981?check_suite_focus=true#step:7:17)). What could be the problem? :thinking:

Reply:
I think this is because of the `See also`. It shouldn't be indented as that is not valid `toml` and thus can't be "lexed".